api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java

```diff
@@ -196,13 +199,24 @@ public void notifySuperUsers(String messageCode, Exception cause, Object... mess
 		alert.setSatisfiedByAny(true);
 		
 		//If there is not user creator for the alert ( because it is being created at start-up )create a user
-		//TODO switch this to use the daemon user when ticket TRUNK-120 is complete
-		if (alert.getCreator() == null) {
-			alert.setCreator(new User(1));
+		if (alert.getCreator() == null) { 
+			TimerSchedulerTask task = new TimerSchedulerTask(new AbstractTask(){ 	
```

Comment:
Does switching to a daemon user require a TimerSchedulerTask?

Reply:
Was I required to just manually get the Daemon user from the DB using its UUID? I thought it was a good practice a Daemon user executing its task on its own (Daemon thread) !

Reply:
Isn't that out of scope considering what the ticket required? "Switch to Daemon user"

Reply:
coool, just in a minute